### PR TITLE
Logging in json fixes, also update dropwizard to 2.0.10

### PR DESCRIPTION
**Pull request by gkazakov111**

### CODE REVIEW

File: src/main/java/com/ft/jerseyhttpwrapper/AttemptLogger.java [hunk 1]
```diff
@@ -73,7 +72,7 @@ public void stop(ResilientClient client, ClientResponse response) {
   }
 
   private void withField(IntermediateYield yield, String key, Object value) {
-    if (isBlank(key) || isNull(value) || isBlank(valueOf(value))) {
+    if (isNull(key) || isNull(value) || isNull(valueOf(value))) {
```
Comment (georgi-denchew) [inline src/main/java/com/ft/jerseyhttpwrapper/AttemptLogger.java:75]: Changing `isBlank` to `isNull` means that we now allow for empty strings to be logged. Is this desired behavior?

**Review by georgi-denchew — CHANGES_REQUESTED** [review]

File: src/main/java/com/ft/jerseyhttpwrapper/AttemptLogger.java (hunk 1 above)
Comment (gkazakov111) [reply src/main/java/com/ft/jerseyhttpwrapper/AttemptLogger.java:75]: it now uses _isBlank_ again

File: src/test/java/com/ft/jerseyhttpwrapper/continuation/ExponentialBackoffContinuationSessionTest.java [hunk 2]
```diff
@@ -98,7 +98,7 @@ public void shouldBackOffExponentially() {
     // interrupt the the thread almost immediately
     threadUnderTest.start();
 
-    sleepFor(5);
+    sleepFor(15);
```
Comment (georgi-denchew) [inline src/test/java/com/ft/jerseyhttpwrapper/continuation/ExponentialBackoffContinuationSessionTest.java:101]: What's the reason behind changing this value?

File: src/test/java/com/ft/jerseyhttpwrapper/ResilientClientTest.java [hunk 3]
```diff
@@ -126,7 +126,7 @@ public void shouldHonourDefaultTimeout() {
         aResponse()
             .withStatus(200)
             .withHeader(CONTENT_TYPE, "text/plain")
-            .withFixedDelay(501)
+            .withFixedDelay(510)
```
Comment (georgi-denchew) [inline src/test/java/com/ft/jerseyhttpwrapper/ResilientClientTest.java:129]: What's the reason behind changing this value?

**Review by georgi-denchew — CHANGES_REQUESTED** [review]

File: src/test/java/com/ft/jerseyhttpwrapper/continuation/ExponentialBackoffContinuationSessionTest.java (hunk 2 above)
Comment (gkazakov111) [reply src/test/java/com/ft/jerseyhttpwrapper/continuation/ExponentialBackoffContinuationSessionTest.java:101]: this one is ok during circleci build - there is no drama with it

File: src/test/java/com/ft/jerseyhttpwrapper/ResilientClientTest.java (hunk 3 above)
Comment (gkazakov111) [reply src/test/java/com/ft/jerseyhttpwrapper/ResilientClientTest.java:129]: this one is ok during circleci build - there is no drama with it

**Comment by gkazakov111** [comment]
I'll close this PR , because there are some more jar updates (pom) and it is made from an unproper branch (the old one for refactoring)

**Pull request closed without merging**